lib/bigstringaf_stubs.c

```diff
@@ -38,36 +38,36 @@
 void
 bigstringaf_blit_to_bytes(value vsrc, value vsrc_off, value vdst, value vdst_off, value vlen)
 {
-    void *src = ((char *)Caml_ba_data_val(vsrc)) + Long_val(vsrc_off),
-         *dst = ((char *)String_val(vdst))       + Long_val(vdst_off);
-    size_t len = Long_val(vlen);
+    void *src = ((char *)Caml_ba_data_val(vsrc)) + Unsigned_long_val(vsrc_off),
+         *dst = ((char *)String_val(vdst))       + Unsigned_long_val(vdst_off);
+    size_t len = Unsigned_long_val(vlen);
```

Comment:
What does the emitted assembly look like on x86? Is there any benefit to using these? I've never seen these unsigned accessors used out in the wild so I'm skeptical that they're necessary.

Reply:
It doesn't change the assembly, but may avoid warnings about pointer signedness.

Reply:
Also this makes it more obvious that no accidental substraction to the pointer can happen.